configuration/components/loki.libsonnet

```diff
@@ -48,6 +48,7 @@ local defaults = {
     compactor: {
       withLivenessProbe: true,
       withReadinessProbe: true,
+      withAffinity: true,
```

Comment:
Sounds like this could be called `withPodAntiAffinity`, as that's effectively what's added?